portfolio/src/main/webapp/script.js

```diff
@@ -0,0 +1,19 @@
+window.addEventListener('DOMContentLoaded', (event) => {
+
+  var element = document.getElementsByClassName("menu")[0];
+  element.addEventListener("click", toggleActiveSection);
+
+  function toggleActiveSection(tab) {
+      var elems = document.querySelector(".active");
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/Element/querySelector (note: on element, not document) will allow you to scope the query to just your menu, helping perf and making this less brittle